boot/bbai_usb_gadget.sh

```diff
@@ -248,7 +248,7 @@ fi
 
 	if [ ! "x${USB_NETWORK_DISABLED}" = "xyes" ]; then
 		ln -s functions/rndis.usb0 configs/c.1/
```

Comment:
Why is RNDIS still here?

Reply:
Hi Jason,
It is my understanding that the RNDIS devices are there to support Windows hosts.  The Beaglebone boot scripts appear to be setup to create 2 Ethernet interfaces as part of the USB composite gadget, one is intended for Windows hosts (RNDIS) and one is intended for MacOS hosts (was CDC ECM now CDC NCM).  Linux hosts see both types of interfaces.  My pull request was to fix a specific issue with the latest MacOS where ECM was broken when part of a USB composite gadget.  I don't have any Windows hosts so I don't know if CDC-NCM could be used as a substitute for RNDIS as well.
James

Reply:
@jadonk,

I just found a Windows 10 system to test with and can confirm that the RNDIS device is still required for a good out of the box experience on Windows 10.  If you plug in a beagle configured with CDC-NCM and RNDIS Ethernet devices (using the code from this pull request), only the RNDIS device is recognised by a vanilla Windows 10 installation.
If you plug it into a Mac with vanilla MacOS 10.15 only the CDC-NCM device is recognised.
If you plug it into a Linux box, both devices are recognised.

James